server/@types/bapv.d.ts

```diff
@@ -196,7 +196,6 @@ export type VisitSessionData = {
   }
   applicationReference?: string
   visitReference?: string
-  previousVisitReference?: string
```

Comment:
Most of the changes in this PR are actually the result of removing this 🙂 ! No longer needed as `visitReference` stays the same and the work is done by `applicationReference`.